src/rust/engine/src/externs/process.rs

```diff
@@ -30,30 +30,39 @@ impl PyProcessConfigFromEnvironment {
   fn __new__(
     platform: String,
     docker_image: Option<String>,
+    remote_execution: bool,
     remote_execution_extra_platform_properties: Vec<(String, String)>,
   ) -> PyResult<Self> {
     let platform = Platform::try_from(platform).map_err(PyValueError::new_err)?;
+    let execution_strategy = match (docker_image, remote_execution) {
+      (None, true) => Ok(ProcessExecutionStrategy::RemoteExecution),
+      (None, false) => Ok(ProcessExecutionStrategy::Local),
+      (Some(image), false) => Ok(ProcessExecutionStrategy::Docker(image)),
+      (Some(_), true) => Err(PyAssertionError::new_err(
+        "docker_image cannot be set at the same time as remote_execution",
+      )),
```

Comment:
Yet TM.

Reply:
Will the Docker Command Runner be running inside RE though? My understanding is that the RE server will handle Docker entirely for us.

I can rename `docker_image` to something like `local_docker_image` if that would help clarify things? The idea is we have 3 execution strategies: Local, LocalDocker, Remote.

Reply:
> Will the Docker Command Runner be running inside RE though? My understanding is that the RE server will handle Docker entirely for us.

Ah... yea, I suppose that on the `Process` itself, this field won't be set, and it will instead be specified by the platform properties.